### ISSUE TITLE
longtest and utf-8 encoding

### ISSUE THREAD

**Issue by hvelarde** [zestsoftware/zest.releaser#126]
Seems that longtest is not setting the right encoding used on documents (utf-8):

![selection_002](https://cloud.githubusercontent.com/assets/341393/9672297/d3ef6042-526f-11e5-822c-8109862b5fc6.png)

in my browser I have to manually select it, as it opens the tab with **Occidental**.

**Comment by reinout** [comment]
Not everything is (yet) going right with readme/changelog reading and writing if there are non-ascii characters. The python 3 support forced us to explicitly turn everything into unicode upon reading. And not everything is right yet, apparently.

Or is the content of the html right? With only a character set declaration in the html msising? @hvelarde, could you check that?

We're now using the "readme" library that's also used by pypi to render the html, so I think that one is OK.

Checking our code I see that we grab the text of the long description by executing `python setup.py --long-description` on the command line and grabbing the output. I suspect we have to look at that part of the code.

**Comment by mauritsvanrees** [comment]
@hvelarde: is this from a publicly available project so that we can play with it? Or can you put a minimal version of that package somewhere for us to download?

**Comment by hvelarde** [comment]
@reinout I think the declaration is missing... in fact is not an HTML document but just the part that should be inside the body... I guess.

@mauritsvanrees yes, is public: https://github.com/plonegovbr/brasil.gov.agenda

**Comment by reinout** [comment]
So... I assume readme spits out the correct utf-encoded text?

Then it would probably be enough if we added a html tag around it with the correct encoding comment?

**Comment by reinout** [comment]
@mauritsvanrees can you retry the generated .rst with a `<html>` and a meta tag (https://en.wikipedia.org/wiki/Character_encodings_in_HTML#Specifying_the_document.27s_character_encoding) ?

**Comment by mauritsvanrees** [comment]
Yes, this works:

```
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
  </head>
  <body>
...
</body>
</html>
```

So we could add that. Or maybe `readme` should.

Alternatively, we could deprecate `longtest` and encourage people to install restview and use `restview --long-description`.

**Comment by reinout** [comment]
Well, we have to test the quality of the readme and I've always found longtest to be handy. Telling them (=me) to install yet another program... :-)

**Comment by reinout** [comment]
Ok, I've added a patch. @mauritsvanrees, can you take a quick peek?

**Comment by reinout** [comment]
@mauritsvanrees: are you making a release now? Otherwise I'll do it.

**Comment by mauritsvanrees** [comment]
I have released 5.5. Thanks.
